<launch>

	<node pkg="filtre" name="Camera_emulate_Node" type="Camera_emulate_Node.py">
	</node>	



	<node pkg="filtre" name="target_filtre" type="target_filtre.py">
	</node>

	<node pkg="filtre" name="Launcher_control_Node" type="Launcher_control_Node.py">
	</node>	

	<node pkg="filtre" name="RPI-SerialComm" type="RPI-SerialCommNode.py">
	</node>	



</launch>
